web/src/app/features/Dashboard/index.tsx

```diff
@@ -18,6 +18,10 @@ export type DashboardProps = {
 
 const Dashboard: FC<DashboardProps> = ({ workspaceId }) => {
   const t = useT();
+  const { useMeQuery } = useMeFetcher();
+  const { me } = useMeQuery();
+  const lang = me?.lang ?? "en";
```

Comment:
```suggestion
  const lang = useLang();
```
I think this is the common way of getting current lang of the app